docs/source/operations/projections/webmerc.rst

```diff
@@ -49,13 +48,8 @@ Example::
 Parameters
 ################################################################################
 
-.. note:: All parameters for the projection are optional.
+.. note:: Projection does not have any additional parameters. See `EPSG.io  <http://epsg.io/3857>`_
```

Comment:
I think it is better to refer to [Wikipedia](https://en.wikipedia.org/wiki/Web_Mercator) here? The epsg.io site doesn't go into much detail. Also, it is easily mistaken as official outlet of the EPSG (which it is not). It is already mentioned under "Further reading".